pkg/controller/devopscredential/devopscredential_controller.go

```diff
@@ -239,7 +239,6 @@ func (c *Controller) syncHandler(key string) error {
 		if sliceutil.HasString(copySecret.ObjectMeta.Finalizers, devopsv1alpha3.CredentialFinalizerName) {
 			if _, err := c.devopsClient.DeleteCredentialInProject(nsName, secret.Name); err != nil {
 				klog.V(8).Info(err, fmt.Sprintf("failed to delete secret %s in devops", key))
-				return err
```

Comment:
Is it ok to just leave the secrets sit in the Jenkins? If that's the case, please add more comments to explain why we do this